website/source/guides/sentinel-policy.html.markdown

```diff
@@ -0,0 +1,207 @@
+---
+layout: "guides"
+page_title: "Sentinel Policies"
+sidebar_current: "guides-sentinel"
+description: |-
+ Nomad integrates with Sentinel for fine grained policy enforcement. Sentinel allows operators to express their policies as code, and have their policies automatically enforced. This allows operators to define a "sandbox" and restrict actions to only those compliant with policy. The Sentinel integration builds on the ACL System.
+---
+
+# Sentinel Policies
+
+Nomad integrates with Sentinel for fine grained policy enforcement. Sentinel allows operators to express their policies as code, and have their policies automatically enforced. This allows operators to define a "sandbox" and restrict actions to only those compliant with policy. The Sentinel integration builds on the [ACL System](/guides/acl.html).
```

Comment:
what is Sentinel? could only find a PHP framework by that name :) maybe a link to the Sentinel service?